README.md

```diff
@@ -36,7 +36,7 @@ yarn install
 ## Build
 
 ```bash
-lerna run build
+npx lerna run build
```

Comment:
can we use `yarn` instead of `npx` as it's the main package manager on the repo?
```suggestion
yarn lerna run build
```